### PR TITLE
chore(melange): update Melange pin

### PR DESCRIPTION
**Pull request by anmonteiro**
The new commit sha introduces fewer dependencies, as `melange` and `mel` (which depends on luv -- slower to compile) are now different packages. We just need to depend on `melange`.

Signed-off-by: Antonio Nuno Monteiro <anmonteiro@gmail.com>

### CODE REVIEW

**Comment by rgrinberg** [comment]
Should we update the melange flake on the nix side as well?

**Comment by anmonteiro** [comment]
Ah yeah, let me do that.

File: flake.lock [hunk 1]
```diff
@@ -265,6 +266,22 @@
...
+      "locked": {
+        "lastModified": 1662774800,
+        "narHash": "sha256-1Rd2eohGUw/s1tfvkepeYpg8kCEXiIot0RijapUjAkE=",
+        "ref": "refs/heads/master",
+        "rev": "bb3a2d3dc52ff0253fb9c2812bd7aa2da03e0fea",
+        "revCount": 1072,
+        "type": "git",
+        "url": "https://gitlab.haskell.org/bgamari/ghc-utils"
+      },
+      "original": {
```
Comment (rgrinberg) [inline flake.lock:280]: How do we know end up depending on this? 😂

Comment (anmonteiro) [reply flake.lock:280]: Huh. inputs of https://github.com/nix-community/dream2nix, which aims to be a generalized `<TOOL>2nix`

**Pull request merged**